Week1/homework/js-exercises/typeChecker.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+// Declare all variables
+let x = 9;
+let y = 67;
+let name = {fist:"manolis", last:"protopsaltis"};
+let color = {color1:"blue", color2:"Red"};
+
+console.log(typeof(x));
+console.log(typeof(y));
+console.log(typeof(name));
+console.log(typeof(color));
+
+if (typeof(x) == typeof(y)){
+    console.log("SAME TYPE");
+}else{
+    console.log("NOT SAME");
+}
+if (typeof(x) == typeof(name)){
+    console.log("SAME TYPE");
+}else{
+    console.log("NOT SAME");
+}
+if (typeof(x) == typeof(color)){
+    console.log("SAME TYPE");
+}else{
+    console.log("NOT SAME");
+}
+if (typeof(y) == typeof(name)){
+    console.log("SAME TYPE");
+}else{
+    console.log("NOT SAME");
+}
+if (typeof(y) == typeof(color)){
+    console.log("SAME TYPE");
+}else{
+    console.log("NOT SAME");
+}
+if (typeof(name) == typeof(color)){
+    console.log("SAME TYPE");
+}else{
+    console.log("NOT SAME");
+}
```

Comment:
Works perfectly!

However, the exercise asks for:
Declare 4 variables: 2 must be  **`strings`** and 2 must be `objects`

In your code, in the two first lines, you declare the following:
`let x = 9;`
`let y = 67;`

Which are type of  `number`.
So, here there is no syntax errors, but there is a logic one...! ;-)